src/services/api.js

```diff
@@ -290,3 +290,42 @@ export const removeComment = comment =>
         Authorization: idJwtToken,
       },
     }));
+
+/* ------ Error Report API functions -------- */
+export const getErrorReports = ({ locationId }) =>
+  axios
+    .request({
+      url: `${config.baseApi}/errorreports`,
+      method: 'get',
+      params: {
+        locationId,
+      },
+    })
+    .then(result => result.data);
+
+export const postErrorReport = ({ locationId, errorReport }) =>
+  getAuthToken() // Remove authorization requirement for posting new error reports?
```

Comment:
Yes, definitely shouldn't require auth.
I believe the backend already doesn't require it.